app/views/movies/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class='new-viewing-button'>
+  <%= button_to "Create Viewing Party for this Movie", "/viewing-party/new" %>
```

Comment:
We may need to use a form_with helper here so as to submit a hidden field with the api-movie-id. Good for now though.